.circleci/config.yml

```diff
@@ -74,6 +81,75 @@ jobs:
           name: "Build"
           command: cond_spot_run_build aztec3-circuits-wasm-linux-clang 64
 
+  aztec3-circuits-wasm-linux-clang-assert:
```

Comment:
Is there any reason all of these should be prefixed with aztec3-circuits, rather than just circuits? Same goes for the directory name

Reply:
aztec3-circuits was already partially being used as a prefix

Reply:
I'll go consistently with circuits, probably a bad judgement call